data/records/totem-datasets-2011.json

```diff
@@ -0,0 +1,195 @@
+        "type": "unsigned int",
+        "unit": "N/A",
+        "variable": "trigger_num"
+      },
+      {
+        "description": "Result of the trigger logic where each bit corresponds to one entry in the trigger menu",
+        "type": "unsigned int",
+        "unit": "N/A",
+        "variable": "input_status_bits"
+      },
+      {
+        "description": "Number of the dataset that this event belongs to",
+        "type": "int",
+        "unit": "N/A",
+        "variable": "dataset_num"
+      },
+      {
+        "description": "Reconstructed interaction point vertex x position based on information from left station",
+        "type": "double",
+        "unit": "Milimeters",
+        "variable": "vtx_x_L"
+      },
+      {
+        "description": "Milimeters; Reconstructed interaction point vertex x position based on information from right station",
+        "type": "double",
+        "unit": "Milimeters",
+        "variable": "vtx_x_R"
+      },
+      {
+        "description": "Reconstructed interaction point vertex x position",
+        "type": "double",
+        "unit": "Milimeters",
+        "variable": "vtx_x"
+      },
+      {
+        "description": "Reconstructed theta collision angle y value based on information from left station",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_y_L"
+      },
+      {
+        "description": "Reconstructed theta collision angle y value based on information from right station",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_y_R"
+      },
+      {
+        "description": "Reconstructed theta collision angle y value",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_y"
+      },
+      {
+        "description": "Reconstructed theta collision angle x value based on information from left station",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_x_L"
+      },
+      {
+        "description": "Reconstructed theta collision angle x value based on information from right station",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_x_R"
+      },
+      {
+        "description": "Reconstructed theta collision angle x value",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th_x"
+      },
+      {
+        "description": "Reconstructed theta collision angle vector magnitude",
+        "type": "double",
+        "unit": "Radians",
+        "variable": "th"
+      },
+      {
+        "description": "Reconstructed four-momentum of the protons after collision",
+        "type": "double",
+        "unit": "GeV^2",
+        "variable": "t"
+      }
+    ],
+    "date_created": [
+      "2011"
+    ],
+    "date_published": "2024",
+    "distribution": {
+      "formats": [
+        "root",
+        "NTuple"
+      ],
+      "number_events": 12606699,
+      "number_files": 1,
```

Comment:
@grzanka Note that your record mentioned 2 files, but actually there was only one `prepared_data.root` file, so I have changed this to 1.

Reply:
Yes, thats correct, we have one file.